Sources/Commands/UserToolchain.swift

```diff
@@ -34,10 +35,16 @@ struct UserToolchain: Toolchain {
 
     init() throws {
         do {
+
             SWIFT_EXEC = getenv("SWIFT_EXEC")
                 // use the swiftc installed alongside ourselves
                 ?? Path.join(Process.arguments[0], "../swiftc").abspath
 
+            lldb = getenv("LLDB")
+                // use the lldb installed alongside ourselves
+                ?? Path.join(Process.arguments[0], "../lldb").abspath
```

Comment:
@ddunbar Can we expect this env var to be set?

AFAIK `Path.join(Process.arguments[0], "../lldb").abspath` will fail, because we are not creating a symlink to `lldb`.